BetaCameras/BaslerToF/BaslerToF.cs

```diff
@@ -332,14 +334,14 @@ protected override void UpdateImpl()
 
                     if (IsChannelActive(ChannelNames.Distance))
                     {
-                        depthImage = new FloatCameraImage(width, height);
+                        distanceImage = new FloatCameraImage(width, height);
                         for (int y = 0; y < height; y++)
                         {
                             for (int x = 0; x < width; x++)
                             {
                                 Coord3D c = bufferPoint3f[y * width + x];
                                 Point3f p = new Point3f(c.x * 0.001f, c.y * 0.001f, c.z * 0.001f);
```

Comment:
Could be written as p = c * 0.001f, or all in one line:
distanceImage[y, x] = (bufferPoint3f[i] * 0.001f).GetLength();

Reply:
done

Reply:
Yeah!